bin/opencollective.js

```diff
@@ -0,0 +1,44 @@
+const child_process = require("child_process");
+const chalk = require("chalk");
+const fs = require("fs");
+const path = require("path");
+
+// Only show emoji on OSx (Windows shell doesn't like them that much ¯\_(ツ)_/¯ )
+function emoji(emoji) {
+	if (process.stdout.isTTY && process.platform === "darwin") {
+		return emoji;
+	} else {
+		return "";
+	}
+}
+
+function print(str = "", color = "dim") {
+	const terminalCols = process.platform === "win32" ? 80 : parseInt(child_process.execSync("tput cols").toString());
+	// eslint-disable-next-line no-control-regex
+	const ansiEscapeSeq = /\u001b\[[0-9]{1,2}m/g;
+	const strLength = str.replace(ansiEscapeSeq, "").length;
+	const leftPaddingLength = Math.floor((terminalCols - strLength) / 2);
+	const leftPadding = " ".repeat(leftPaddingLength);
+	str = chalk[color](str);
+	console.log(leftPadding, str);
```

Comment:
Hi, nice work. Can you add a day check to only output this on mondays?

Reply:
I had some problems with the azure CI but it's done now ;)